card/card.css

```diff
@@ -0,0 +1,171 @@
+@import url("https://cdn.jsdelivr.net/npm/bootstrap-icons@1.7.2/font/bootstrap-icons.css");
+@import url(../styles.css);
+* {
+    font-family: sans-serif;
+}
+:root {
+    --main-box-shadow: 0 3px 10px rgb(0 0 0 / 0.2);
+    --hover-effect-color: #eeeded;
+    --light-gray: rgb(176, 176, 176);
+}
+
+.main-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    /* margin: 4rem; */
+    border: 1px solid red;
+}
+html {
+    font-size: 100%;
+}
+h2 {
+    font-size: 1.5rem;
+}
+h3 {
+    font-size: 1rem;
+}
+p {
+    font-size: 0.9rem;
+}
+
+.header {
+    opacity: 0.9;
+}
+img {
+    height: 100%;
+    width: 100%;
+}
+.component {
+    display: flex;
+    flex-direction: column;
+    height: 21rem;
+    width: 20rem;
+    border-radius: 4px;
+    box-shadow: var(--main-box-shadow);
+    margin: 4rem;
+}
+.horizontal-card{
+    height: 13rem;
+}
+img {
+    height: 13rem;
+}
+.badge-card{
+    position: relative;
+    top: -13rem;
+    height: 1.3rem;
+    width: 3rem;
+    padding: 3px 6px;
+    color: white;
+    border-radius: 3px;
+    background-color: blue;
+}
+.image img {
+    border-radius: 4px 4px 0px 0px;
+}
+.list button {
+    margin-top: 0.4rem;
+    background-color: blue;
+    color: white;
+    padding: 10px 10px;
+    border: none;
+}
+
+.btn-secondary-horizontal {
+    width: 5rem;
+    margin-left: 1rem;
+}
+.main-card-text{
+    margin: 5px;
+}
+.main h2 {
+    font-weight: 400;
+    font-size: 1.6rem;
+}
+.main h3 {
+    font-weight: 400;
+    font-size: 1rem;
+    color: gray;
+    margin-bottom: 1rem;
+}
+.main p {
+    font-size: 0.9rem;
+    font-weight: 300;
+    color: gray;
+}
+.header:hover {
+    opacity: 1;
+    cursor: pointer;
+}
+.btn-text {
+    display: flex;
+}
+.btn-read:hover {
+    background-color: rgb(235, 229, 229);
+}
+.dismiss{
+    color: white;
+    position: relative;
+    top: -12rem;
+    left: 18rem;
+}
+/* .content {
+    position: relative;
+    top: -1.6rem; */
+}
+.btn-font i {
+    height: 2.8rem;
+    width: 2.8rem;
+    text-align: center;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.btn-text i:hover {
+    background-color: rgb(235, 229, 229);
```

Comment:
please use hex code for colors and also use variables for colors are advised by everyone